src/app/armory/selectors.ts

```diff
@@ -0,0 +1,13 @@
+import { currentAccountSelector } from 'app/accounts/selectors';
+import { accountRoute } from 'app/routes';
+import { createSelector } from 'reselect';
+
+/**
+ * Returns the account path parameters
+ */
+export const armoryAccountRouteSelector = createSelector(currentAccountSelector, (account) => {
+  if (!account) {
+    return undefined;
+  }
+  return accountRoute(account);
```

Comment:
It would be good to build the whole route here but we would need to store the armoury item details in the store, which I believe we don't do at this stage.